string name
float64[] b			# A dq < b
std_msgs/Float64MultiArray A	# A dq < b
